ci/wheel_smoke_test_distributed_ucxx.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+
+import pytest
+
+from distributed.comm import connect, listen
+from distributed.protocol import to_serialize
+
+import ucxx
+
+import distributed_ucxx  # noqa: E402
```

Comment:
```suggestion
import distributed_ucxx
```

Reply:
Removed in https://github.com/rapidsai/ucxx/pull/167/commits/d6af7c692fa20db8706fbe26c8ad7b647bd3ddbc